doc/md/language-manual.md

```diff
@@ -83,10 +83,12 @@ All comments are treated as whitespace.
 The following keywords are reserved and may not be used as identifiers:
 
 ``` bnf
-actor and assert async async* await await* break case catch class continue debug
-debug_show do else flexible false for from_candid func if ignore import
-in module not null object or label let loop private public query return
-shared stable switch system throw to_candid true try type var while with
+
+actor and assert async async* await await* break case catch class
+composite continue debug debug_show do else flexible false for
```

Comment:
Including `composite` as reserved keyword impacts backward compatibility (former identifiers named `composite` become invalid). This would technically be a breaking change, isn't it?

Reply:
Yes, it is, but probably won't brake many. Maybe we should bump to 0.10?

Reply:
Another alternative would be to use `composite_query` (ugly IMO) or `query*` (confusing, subtle IMO)

Reply:
I think `composite query` is much nicer to read. (Another possibility would be to introduce non-reserved keywords but the complexity is not worth it.) I would keep the current syntax.